README.md

```diff
@@ -102,25 +102,43 @@ Do not try to set `fzf_fish_custom_keybindings` in your `config.fish` because th
+To make fzf's interface friendlier, `fzf.fish` takes the liberty of setting a sane `FZF_DEFAULT_OPTS` if it is not already set. See [conf.d/fzf.fish][] for more details. This affects fzf even outside of this plugin. If you would like to remove this side effect or just want to customize fzf's default options, then set export your own `FZF_DEFAULT_OPTS` variable. For example:
 
 ```fish
-set --universal --export FZF_DEFAULT_OPTS --height 50% --margin 1
+set --export FZF_DEFAULT_OPTS --height 50% --no-extended +i
 ```
 
-Alternatively, you can override it in your `config.fish`:
+### Pass fzf options to a specific command
 
-```fish
-set --export FZF_DEFAULT_OPTS --height 50% --margin 1
-```
+Custom options can be passed to fzf for each feature individually through these variables:
+
+| Feature                | Variable              |
+| ---------------------- | --------------------- |
+| Search directory       | `fzf_dir_opts`        |
+| Search git status      | `fzf_git_status_opts` |
+| Search git log         | `fzf_git_log_opts`    |
+| Search command history | `fzf_history_opts`    |
+| Search shell variables | `fzf_shell_vars_opts` |
+
+These variables are appended last to fzf's argument list. Because fzf will use the option appearing last when options conflict, your custom options can override preset options. This unlocks a variety of possibilities in customizing and augmenting each feature such as:
+
+- add [key bindings](https://www.mankier.com/1/fzf#Key/Event_Bindings) within fzf to operate on the selected line:
+  - [open file in Vim](https://github.com/junegunn/fzf/issues/1360)
+  - [preview image files](https://gitter.im/junegunn/fzf?at=5947962ef6a78eab48620792)
+  - [copy to clipboard](https://betterprogramming.pub/boost-your-command-line-productivity-with-fuzzy-finder-985aa162ba5d)
+  - git checkout commit
+  - git reset file
+- adjust the preview command or window
+- [re-populate fzf's input list on demand](https://github.com/junegunn/fzf/issues/1750)
+- change the search mode
 
 ### Change the command used to preview folders
 
-The search files feature, by default, uses `ls` to preview the contents of a directory. To integrate with the variety of `ls` replacements available, the command used to preview directories is configurable through the `fzf_preview_dir_cmd` variable. For example, in your `config.fish`, you may put:
+The search directory feature, by default, uses `ls` to preview the contents of a directory. To integrate with the variety of `ls` replacements available, the command used to preview directories is configurable through the `fzf_preview_dir_cmd` variable. For example, in your `config.fish`, you may put:
```

Comment:
Finally settling on the name search directory to make room for search files by pattern (#118) in the features namespace